pkg/kubelet/network/cni/cni.go

```diff
@@ -271,7 +272,11 @@ func (network *cniNetwork) deleteFromNetwork(podName string, podNamespace string
 	err = cninet.DelNetwork(netconf, rt)
 	if err != nil {
 		glog.Errorf("Error deleting network: %v", err)
-		return err
+
+		subStr := "no such file or directory"
```

Comment:
A big 'ol fat comment here detailing what is going on here would be helpful.  This obviously smells bad.

We should also commit to removing this soon after 1.6 and fixing it "the right way".  We can be more nuanced there.